swagger.yaml

```diff
@@ -4041,6 +4087,18 @@ definitions:
         format: date-time
         x-go-custom-tag: gorm:"type:timestamp with time zone"
         description: Time at which the operator was last updated.
+      bundles:
+        type: array
+        description: List of bundles associated with the operator. Can be empty.
+        items:
+          type: string
+        x-go-custom-tag: gorm:"type:text[]"
+        x-go-type:
+          type: StringArray
+          import:
+            package: github.com/lib/pq
+          hints:
+            noValidation: true
```

Comment:
What is the reason to have this list for each cluster? Can't that be inferred from the list of operators of each bundle?

I assume you are doing this to store somewhere the list of bundles that are enabled in a particular cluster, but wouldn't it be more straightforward to have a separate table (similar to the `monitored_operators` table) to store this explicitly? I mean something like this:

```sql
create table enabled_operator_bundles (
  cluster_id text not null,
  bundle_name text not null,
  primary key (cluster_id, bundle_name)
)
```

Reply:
Hmm, the reason we are doing this is because that's what the ticket asked.
"adding a list of bundle labels to each operator"
Moreover, we don't keep track of bundles that are enabled in a particular cluster.
I added the bundles API endpoints because it makes a bit more sense to me (and the UI dev) 
I agree that we could keep a map with the bundle to operator list mapping and it should be just fine (for the bundles API endpoint) but than you won't get the list of bundles the operator belongs to when you get the operator.
Overall, the ticket asked to add bundles list to the operator I did the extra step of exposing what I belive the UI cares about in a dedicated endpoint.

Reply:
OK, I get it, thanks for explaining.